src/NServiceBus.Core/Transports/Msmq/MsmqTransport.cs

```diff
@@ -33,9 +33,13 @@ public override TransportInfrastructure Initialize(SettingsHolder settings, stri
             var msmqSettings = connectionString != null ? new MsmqConnectionStringBuilder(connectionString)
                 .RetrieveSettings() : new MsmqSettings();
 
+            msmqSettings.UseDeadLetterQueueForMessagesWithTimeToReachQueue = settings.GetOrDefault<bool>(UseDeadLetterQueueForMessagesWithTimeToReachQueue);
```

Comment:
Would it make sense to explicitly set the default instead of doing GetOrDefault ? Might be easier to understand and spot.